package.json

```diff
@@ -34,9 +34,11 @@
     "css-loader": "^6.8.1",
     "dart-sass": "^1.25.0",
     "eslint": "^8.54.0",
+    "eslint-config-prettier": "^9.1.0",
```

Comment:
Add prettier, the plugin to socket it in, and the config that prevents conflicts with eslint.

Essentially we keep eslint as a way to find and prevent bugs, while prettier is soley focused on code styling per:
https://prettier.io/docs/en/comparison